filelist.csv

```diff
@@ -206,3 +206,9 @@ g2012bp0.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:6319,EPSG:6641,,us_noa
 g2018u0.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:6319,EPSG:5703,,us_noaa,NOAA,Public Domain
 g2018p0.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:6319,EPSG:6641,,us_noaa,NOAA,Public Domain
 100800401.gsb,HORIZONTAL_OFFSET,,EPSG:4230,EPSG:4258,,es_cat_icgc,Institut Cartogràfic i Geològic de Catalunya (ICGC),Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
+ISN93_ISN2016.gsb,HORIZONTAL_OFFSET,,EPSG:4659,EPSG:8086,,lmi,National Land Survey of Iceland,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
+ISN2004_ISN2016.gsb,HORIZONTAL_OFFSET,,EPSG:5324,EPSG:8086,,lmi,National Land Survey of Iceland,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
+Icegeoid_ISN2004.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:5323,EPSG:8089,,lmi,National Land Survey of Iceland,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
+Icegeoid_ISN93.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:4945,EPSG:8089,,lmi,National Land Survey of Iceland,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
+Icegeoid_ISN2016.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:8085,EPSG:8089,,lmi,National Land Survey of Iceland,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
+ISN_vel_beta.ct2 + ISN_vel_beta.gtx,VELOCITY_MODEL,,,,,lmi,National Land Survey of Iceland,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
```

Comment:
for completness could you add records like:
ISN93_ISN2016.gsb + ISN93_ISN2016.gtx,DEFORMATION_MODEL ... 
ISN2004_ISN2016.gsb + ISN2004_ISN2016.gtx,DEFORMATION_MODEL ...